image-layout.md

```diff
@@ -115,4 +116,42 @@ $ cat ./blobs/sha256/e692418e4cbaf90ca69d05a66403747baa33ee08806650b51fab815ad7f
 [tar stream]
 ```
 
+## Index 
+
+Index is intended to be generated from the content in an image-layout after any updates have been made.
```

Comment:
But there are updates that are not included in “ref additions or removals” (e.g. blob addition, or changing a ref's descriptor payload) which would *not* require index regeneration.  I'm suggesting we make our regeneration suggestions as narrowly-scoped as possible, because conflicts around regenerating this index are [one of the major pain-points with this approach][1].

[1]: https://github.com/opencontainers/image-spec/pull/438#issuecomment-258199793